README.md

```diff
@@ -88,9 +88,10 @@ var options = {
 };
 ```
 
-#### `uncaughtException` (*Function*, optional)
+#### `uncaughtException` (*Function*, optional) **DEPRECATED**
 Handler for `uncaughtException` errors. See the [endgame](https://github.com/totherik/endgame) module for defaults.
 
+**Deprecated**: See `shutdown` middleware instead.
```

Comment:
Yeah - or maybe it doesn't need to be deprecated at all. It could still be a fallback for error that happen outside the middleware chain.